spec/support/mixins/pg_environment_updater.rb

```diff
@@ -22,6 +22,33 @@ def create_postgres_role
   end
   module_function :create_postgres_role
 
+  # This is a CI only function that will create the manageiq postgres config
+  # directory at /etc/manageiq/postgresql.conf.d.
+  #
+  # This is fine to do on CI as it is an ephemeral VM, so it isn't going to
+  # mess with state at all of a persisted machine (like creating this on
+  # someone's dev box would).
+  #
+  # This does not add in all of the configs that you would find on an
+  # appliance, however, and just defines enough to get things working (so
+  # basically just 'ssl = on').
+  #
+  # See PostgresRunner#fix_pg_conf for how we handle this locally.
+  def create_stub_manageiq_configs_on_ci
```

Comment:
Why are we doing this? Does the restore for the base backup fail if this doesn't exist? Or is this just to be as close to prod as possible?

Reply:
> Does the restore for the base backup fail if this doesn't exist? Or is this just to be as close to prod as possible?

Both.  `postgres` won't even start up if this directory doesn't exist, and ideally I was trying to have this match prod as much as possible (but again, this seems like it was a very futile effort).

Reply:
Yeah, I think we should resolve this by not using a backup from an appliance to test restore.